src/app/InteractionModelEngine.h

```diff
@@ -617,6 +632,13 @@ CHIP_ERROR ReadSingleClusterData(const Access::SubjectDescriptor & aSubjectDescr
                                  const ConcreteReadAttributePath & aPath, AttributeReportIBs::Builder & aAttributeReports,
                                  AttributeValueEncoder::AttributeEncodeState * apEncoderState);
 
+/**
+ *  Check whether concreate attribute path is in ember storage.
+ *  @param[in]    aPath                 The concrete path of the data being read.
+ *  @retval  boolean
+ */
+bool ConcreteAttributePathExists(const AttributePathParams & aPath);
```

Comment:
Also, shouldn't this take a `ConcreteAttributePath`?  Or at least document that this should only be called if the `AttributePathParams` represents a concrete path or something?